### DISCUSSION TITLE
notification and other features

### DISCUSSION THREAD

**Discussion by matheusbranhann** [KevinBytesTheDust/CVAmp#240] [General]
I come to ask as a supporter for some features.

Place a color notification on the instance when viewing an ad, color suggestion: purple:
When the instance is detected by Twitch, it will also have some color.
also show in info which proxy the instance is currently using
the fact of the proxy, is to know if when generating an instance it will automatically be detected because of the proxy

**Comment by jlplenio** [comment]
A marker for when the instance is detected is a great idea @ConnorMAD.
As I do not condone anything regarding ads, I will not add this feature.
